docs/README.md

```diff
@@ -4,6 +4,17 @@
 * [Gardener landing page `gardener.cloud`](https://gardener.cloud/)
 * ["Gardener, the Kubernetes Botanist" blog on kubernetes.io](https://kubernetes.io/blog/2018/05/17/gardener/)
 
+## Proposals
```

Comment:
I feel the order of sections could be improved, how about this order :

Overview (currently there is no 2nd level section for the first 3 topics)
Concepts
Usage
Proposals
Development
Extensions
Testing
Deployment